Resources/Prototypes/_Goobstation/Entities/Objects/Misc/rubber_stamp.yml

```
@@ -10,3 +10,205 @@
   - type: Sprite
     sprite: _Goobstation/Objects/Misc/stamps.rsi
     state: stamp-nanorep
+
+- type: entity
+  name: cybersun industries rubber stamp
+  parent: RubberStampBase
+  id: RubberStampCybersun
+  components:
+  - type: Stamp
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)